### ISSUE TITLE
How to add existing files to uploader.

### ISSUE THREAD

**Issue by irjayjay** [alexsasharegan/vue-transmit#35]
As per your request, I'm adding issues as I get stuck.

I've got a list of successfully uploaded files coming from the server in JSON format.
I'd like to display these to the user in the same list as the other uploaded files.
I already have an ajax function that fires on delete, so I'm not too concerned about the uploader handling the delete functionality.

Is there a way to access an addFile() method on the uploader?

**Comment by alexsasharegan** [comment]
Adding files is only for adding files you want to upload. You can add a file using a native file object via the `addFile` method.

I  think it might be a common misunderstanding, but Vue Transmit isn't supposed to hold your state for files, it just holds a queue of files you want to upload (and uploads them). You should use the success event to capture files into your state management (local state, Vuex, etc.).

In other words, one of your components should aggregate files from your backend and from successful uploads from Vue Transmit.

**Comment by alexsasharegan** [comment]
Let me know if that makes sense! I know it might sound like a non answer, but it's a better architecture in the end.
